src/components/LandingPage/LandingPage.js

```diff
@@ -1,11 +1,51 @@
-import React from "react";
+import background from "./bg-landingPage.jpg";
+import { Card } from "react-bootstrap";
+import styles from "./LandingPage.module.css";
+import "bootstrap/dist/css/bootstrap.min.css";
 
 const LandingPage = () => {
-  return (
-    <div style={{ height: "100vh" }}>
-      <div>LandingPage</div>
-    </div>
-  );
+	return (
+		<>
+			<div
+				style={{
+					backgroundImage: `url(${background})`,
+					backgroundRepeat: "no-repeat",
+					backgroundPosition: "center",
+					backgroundSize: "cover",
+					height: "100vh",
+					overflowY: "auto",
+				}}>
```

Comment:
I'm sorry but you haven't solved the problem. What is the purpose of setting overflowY to auto? Maybe you need to use a different value here?

Reply:
It was by mistake when I tried to see a double scroll